internal/pipe/ko/ko_test.go

```diff
@@ -166,20 +214,89 @@ func TestPublishPipeSuccess(t *testing.T) {
 				},
 				Kos: []config.Ko{
 					{
-						ID:         "default",
-						Build:      "foo",
-						WorkingDir: "./testdata/app/",
-						BaseImage:  table.BaseImage,
-						Repository: fmt.Sprintf("%s/goreleasertest", registry),
```

Comment:
> I tried to use `Bare: true` for the tests to make the image reference easier to build, but I get an error

Found the issue, there is a double slash in the `Repository`, `registry` already contains a trailing slash, this is going to be fixed